benchmarks/dynamo/torchbench.py

```diff
@@ -54,6 +55,13 @@ def setup_torchbench_cwd():
     "densenet121": 4,
     "hf_Reformer": 4,
     "timm_efficientdet": 1,
+
+    # TODO: what's the better way to override batch size.
+    # Will the change here affect other scripts?
```

Comment:
Can we put these batch size changes behind a command line flag?  You should be able to modify the place this var is used.

Reply:
yep, will use the '--batch_size' option. I lent my XLA dev environment to brian to fix the max2pool issue ATM. Will update the PR once I get that back